_site/nips-2018-es.html

```diff
@@ -92,7 +91,13 @@ <h3>Fechas Importantes</h3>
         <h2>Instrucciones de sumisión</h2>
         <p>Recomendamos a estudiantes, post-docs e investigadores en todas las áreas de inteligencia artificial que se identifican primordialmente como LatinX a presentar un resumen técnico mencionando publicaciones nuevas, previas, o que están siendo revisadas. Invitamos a los autores a que envíen resúmenes en las áreas de teoría, metodología, así como sus aplicaciones. Los resúmenes pueden referirse a trabajos terminados o en progreso. Dado que no es necesario que el primer autor sea quien presente el trabajo, recomendamos a los autores que resalten la contribución de las personas LatinX en el resúmen, especialmente en el caso que sean ellos quienes presenten el trabajo.</p>
         <p>En general, los trabajos aceptados serán presentados en una sesión de pósters. Habrá un número reducido de trabajos que serán seleccionados para que sus autores hagan una presentación oral de 15 minutos. Las propuestas serán revisadas por voluntarios de la comunidad en un proceso doblemente-ciego. Invitamos a que los autores participen como voluntarios para revisar las propuestas para LatinX en IA, indicándolo así durante el proceso de envío de sus resúmenes.</p>
-        <a class="project-link" href="/nips-2018-apply-es">Ingreso de resumen</a>
+
+	<p> 
```

Comment:
@yanneta this section should be formatted with an <h3> and two <p> tags to match the English version of this page. Please resubmit with changes. Thank you.